README.md

```diff
@@ -104,6 +104,22 @@ Example:
 
     docker build --build-arg flags="-y -b -s -a https://github.com/OSEHRA/vxVistA-M/archive/master.zip" -t vxvista .
 
+##### available flags
```

Comment:
not sure that all autoInstaller flags should be here. It is just one more place that needs to be updated if more options are added in the future. Help is generated by the autoinstaller script. maybe just reference that section.

Though the specific option for doing extraction should be called out in this document - How to extract M components from a CACHE.DAT file.